deploy/crds/v1/policy.open-cluster-management.io_iampolicies.yaml

```diff
@@ -32,6 +32,15 @@ spec:
           spec:
             description: IamPolicySpec defines the desired state of IamPolicy
             properties:
+              ignoreClusterRoleBindings:
```

Comment:
I noticed that we have 2 copies of this CRD - one here at `deploy/crds/v1/policy.open-cluster-management.io_iampolicies.yaml` and another at `deploy/crds/policy.open-cluster-management.io_iampolicies_crd.yaml`.

The `deploy` make target runs `kubectl apply -f deploy/crds/ ...` so I don't think that will pick up this change. But the `install-crds` make target does apply this (changed) crd yaml...

tl;dr, can we remove whatever the extra CRD YAML is, and ensure the make targets still work. I think it's possible to address this in another PR if you'd prefer.

Reply:
This reminds me that any CRD change on managed cluster side needs to make into https://github.com/open-cluster-management/klusterlet-addon-operator/blob/main/helm-charts

Reply:
After discussing this with @JustinKuli, we decided to also include the change in the v1beta1 CRD rather than deleting it.

Reply:
@ycao56 I'll make the change there as well if this PR gets approved.